### PR TITLE
fix: User and space avatar are not accessible - EXO-62527 - Meeds-io/meeds#1356

### PR DESCRIPTION
**Pull request by akhanfir**
Before this change, when create an activity and refresh stream, the a tag of avatar have not a discernible name, the img tag for avatar have a role=presentation which is not the correct one and space and user avatar are concerned. To fix this, added:
  -the arial-label for this a tag of user avatar by the key popover.userAvatar.title which returns a text Open {userFullname} profile.
  -the alt for this image tag by the key popover.userAvatar.title which returns a text{userFullname}'s avatar.
  -the arial-label for this a tag of space avatar by the key space.logo.banner.title which returns a text Open {userFullname} profile.
  -the alt for this image tag by the keyspace.logo.banner.alt which returns a text{userFullname}'s avatar.
After this change, user and space avatar are accessible.

### CODE REVIEW

**Review by rdenarie — DISMISSED** [review]
The task speaks about user and spave avatar.
I see that you treat user avatar (OK), and Space Banner (potentially ok).
But you didnt treat space avatar

**Review by rdenarie — COMMENTED** [review]
In addition, if you treat space banner, you should treat user banner also

**Comment by akhanfir** [comment]
In this PR i just treated space avatar and user avatar as requested in the task neither space banner nor user banner.
Should i treat them in this PR or in a separate task ?

**Comment by rdenarie** [comment]
> In this PR i just treated space avatar and user avatar as requested in the task neither space banner nor user banner. Should i treat them in this PR or in a separate task ?

Your treat user avatar and space banner as shown in modified file of the pr 
image

**Comment by akhanfir** [comment]
the file ExoSpaceLogoBanner.vue contains the code of the dropdown menu of the space avatar on on the site topbar which contains the space avatar not the space banner 
image

**Review by rdenarie — COMMENTED** [review]
Ok, so you didnt treat SpaceAvatar.vue

The task speaks about user and spaces avatar in activity stream mainly.

So, it is a good point to have treaten the space avatar in the top bar, but you also need to do it for space avatar in acitvities

File: webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
-          <a :href="portalPath">
+          <a :href="portalPath" :aria-label="$t('space.logo.banner.title',{0:logoTitle})">
             <v-list-item-avatar 
               v-if="logoPath"
               id="UserHomePortalLink"
               size="30"
               class="tile my-0 spaceAvatar ms-0 me-3"
               tile>
-              <v-img :src="logoPath" :alt="logoTitle" />
+              <img :src="logoPath" :alt="$t('space.logo.banner.alt',{0:logoTitle})">
```
Comment (rdenarie) [inline webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue]: Why did you change to img instead of v-img ?

Comment (akhanfir) [reply webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue]: When converting the view tag to html tag the alt props will be converted to arial-label and in the task it asks me to add an alt props in image tag

**Comment by akhanfir** [comment]
about space avatar in activities I created a PR in exoplatform/news https://github.com/exoplatform/news/pull/999/files

File: webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue (hunk 1 above)
Comment (rdenarie) [reply webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue]: If you look at this example :
https://play.vuetifyjs.com/#eNpVUEGOwjAM/Erk8xKz6molqoLEOygHK4QSlLRWYgoI8XfSAipcEs+MZc94c4MUDa6ZdX+yUEIlNrAnsau6VarqZ8Q8liMwXSvkWhtf1Ei60LyRUuXZ7eSwrKGYz2uYeEpsjcwiieuy+vuPi0/ZdL2NE8yectNBhFOJaHbt4E7c/npM2nQBXaDGJmSK5D1dMGTD0ZHXR26+tnrJc/KrxF5kUlYVjr7fyTL6jjYwz+QVflwE7j9Q6EIvYPj/dAHbB5c8YQg=

you can see that v-img with alt parameter generates a alt attribute, not a aria-label one

**Comment by rdenarie** [comment]
> about space avatar in activities I created a PR in exoplatform/news https://github.com/exoplatform/news/pull/999/files

Why do you change only in news ? 
Space avatar are also used in activities, in task, in project ... every where in the platform, from the component SpaceAvatar.vue.

You should change this one

**Comment by Jihed525** [comment]
@rdenarie is it ok ?

**Comment by rdenarie** [comment]
I remove my request change to let @ahamdi review and validate this PR without blocking on my side

**Review by ahamdi — APPROVED** [review]
